shell.nix

```diff
@@ -1,17 +1,17 @@
-{ nixpkgs ? import <nixpkgs> {}, compiler ? "ghc7101" }:
+{ nixpkgs ? import <nixpkgs> {}, compiler ? "default" }:
 
 let
 
   inherit (nixpkgs) pkgs;
 
-  f = { mkDerivation, aeson, aeson-pretty, amazonka
-      , amazonka-autoscaling, amazonka-core, amazonka-ec2, amazonka-elb
-      , amazonka-route53, async, attoparsec, base, base64-bytestring
-      , bytestring, conduit, conduit-extra, containers, directory
-      , exceptions, filepath, lens, lifted-base, mtl, natural-sort
-      , optparse-applicative, pretty-show, process, random, resourcet
-      , scientific, semigroups, stdenv, tagged, text, time, unix
-      , unordered-containers, vector, vk-posix-pty, xml-conduit
+  f = { mkDerivation, aeson, aeson-pretty, amazonka, amazonka-core
```

Comment:
you probably want to include this in a separate PR (or remove it, as expressions are now in ug)